plugin/src/api/index.ts

```diff
@@ -30,12 +30,57 @@ export class TodoistApiClient {
     }
   }
 
-  public async createTask(content: string, options?: CreateTaskParams): Promise<void> {
-    const body = snakify({
-      content: content,
-      ...(options ?? {}),
+  public async createTask(content: string, options?: CreateTaskParams, useQuickAddEndpoint?: boolean): Promise<void> {
+    if (useQuickAddEndpoint) {
+      // Quick add endpoint expects a different payload
+      const body = new URLSearchParams({
+        text: content,
+        ...((options?.description ? { description: options.description } : {})),
+        ...((options?.projectId ? { project_id: options.projectId } : {})),
+        ...((options?.sectionId ? { section_id: options.sectionId } : {})),
+        ...((options?.dueDate ? { due_date: options.dueDate } : {})),
+        ...((options?.dueDatetime ? { due_datetime: options.dueDatetime } : {})),
+        ...((options?.labels ? { labels: options.labels.join(",") } : {})),
+        ...((options?.priority ? { priority: String(options.priority) } : {})),
+      });
+      await this.doQuickAdd("https://api.todoist.com/sync/v9/quick/add", body);
```

Comment:
The sync API is deprecated and will be removed towards the end of the year, it looks like there is a [quick add endpoint](https://developer.todoist.com/api/v1#tag/Tasks/operation/quick_add_api_v1_tasks_quick_post) on the new API, but it looks slightly different, can we pivot onto that one?